app/assets/v2/js/shared.js

```diff
@@ -438,6 +438,54 @@ window.addEventListener('load', function() {
                 });
             };
         }
+        if($("#faucet_form").length){
```

Comment:
does this need to be in shared.js?  can it just be in a page-specific js?

Reply:
as we are already doing a setTimeout in shared to check form status, I wanted to use that instead of running another setTimeout